packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -416,10 +421,22 @@ export class UnfurlService extends BaseService {
         // eslint-disable-next-line no-param-reassign
         retries -= 1;
 
-        return wrapSentryTransaction(
-            'UnfurlService.saveScreenshot',
-            {},
+        return Sentry.startSpan(
+            {
+                name: 'UnfurlService.saveScreenshot',
+                op: 'saveScreenshot',
+            },
             async (span) => {
+                Sentry.setTags({
+                    'page.type': lightdashPage ?? 'undefined',
```

Comment:
tags need to be like this for them to be set on the issue level 'TAG'

Reply:
@joaoviana it needs to be a string `"undefined"`?

Reply:
otherwise the tag isn't sent so I would rather have it set to undefined, hope that's ok!